### PR TITLE
Added EmergingIssues API paths from 2018-07-01.

### PR DESCRIPTION
**Pull request by abhishek-corp**
## ARM API Information (Control Plane)

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow.
</i>
Azure 1st Party Service can try out the [Shift Left](https://aka.ms/ShiftLeft) experience to initiate API design review from ADO code repo.  If you are interested, may request engineering support by filling in with the form https://aka.ms/ShiftLeftSupportForm.

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month. ```This is existing API (from 2018), adding swagger for completeness.```
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month. ```Dec 2022```
  4. By default, Azure SDKs of all languages (.NET/Python/Java/JavaScript for both management-plane SDK and data-plane SDK, Go for management-plane SDK only ) MUST be refreshed with/after swagger of new version is published. If you prefer NOT to refresh any specific SDK language upon swagger updates in the current PR, please leave details with justification here.

### Contribution checklist (MS Employees Only):
- [x] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

> **Applicability**: :warning:
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs
> - Adding new properties
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them.

### Breaking Change Review Checklist
If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by abhishek-corp** [comment]
These APIs exist already and have been captured in 2018-07-01 swagger version. Adding it here (2022-10-01-preview) for swagger completeness.

```/providers/Microsoft.ResourceHealth/emergingIssues``` : https://github.com/Azure/azure-rest-api-specs/blob/639376b2bf9f0f36debfd7fce7debdf7b72578af/specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2018-07-01/ResourceHealth.json#L386

```/providers/Microsoft.ResourceHealth/emergingIssues/{issueName}```: https://github.com/Azure/azure-rest-api-specs/blob/639376b2bf9f0f36debfd7fce7debdf7b72578af/specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2018-07-01/ResourceHealth.json#L350

**Comment by abhishek-corp** [comment]
This PR didn't introduce any breaking change. Some additional property  'systemData' was removed from common-types, and this is affecting this PR.

We did not make any changes to lines highlighted in breaking change check.

**Comment by abhishek-corp** [comment]
LintDiff error explanation

>the default error response schema does not correspond to the schema documented at https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#error-response-content.
Location: [Microsoft.ResourceHealth/preview/2022-10-01-preview/ResourceHealth.json#L791](https://github.com/Azure/azure-rest-api-specs/blob/f3807fadd888be7b6da3ac5d6e7058b2b185dd7b/specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2022-10-01-preview/ResourceHealth.json#L791)

>the default error response schema does not correspond to the schema documented at https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#error-response-content.
Location: [Microsoft.ResourceHealth/preview/2022-10-01-preview/ResourceHealth.json#L752](https://github.com/Azure/azure-rest-api-specs/blob/f3807fadd888be7b6da3ac5d6e7058b2b185dd7b/specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2022-10-01-preview/ResourceHealth.json#L752)

>>As a service, we do not want to break expected behavior and want to keep parity with other APIs and older version (2018-07-01). Hence, we're not adopting recommended ErrorResponse. We have discussed this with ARM team members on emails.
This has already been brought up while publishing 2022-05-01-preview version for the first time and resolved then.
https://github.com/Azure/azure-rest-api-specs/pull/21286

**Review by srinathgrs — APPROVED** [review]

**Comment by abhishek-corp** [comment]
LintDiff error explanation

The top-level resource 'emergingIssuesGetResult' does not have list by subscription operation, please add it.
Location: [Microsoft.ResourceHealth/preview/2022-10-01-preview/ResourceHealth.json#L1683](https://github.com/Azure/azure-rest-api-specs/blob/ca70dd283e6d7a36ec4c12639db9f11bcaf0c8b2/specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2022-10-01-preview/ResourceHealth.json#L1683)

>> This is an existing API, it is only supported for Tenants. We do not support it for subscriptions. Adding this API path for swagger completeness only, I'm not introducing new feature.

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2022-10-01-preview/ResourceHealth.json [hunk 1]
```diff
@@ -1591,6 +1663,156 @@
...
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/emergingIssue",
+          "description": "The emerging issue entity properties.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
```
Comment (raosuhas) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2022-10-01-preview/ResourceHealth.json:1695]: >/ [](http://example.com/codeflow?start=69&length=1)

v3 ?

**Comment by raosuhas** [comment]
Added minor comment to use v3 types and signing off. please fix before checking in

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2022-10-01-preview/ResourceHealth.json (hunk 1 above)
Comment (abhishek-corp) [reply specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2022-10-01-preview/ResourceHealth.json]: fixed.

**Review by live1206 — APPROVED** [review]

**Pull request merged**